package.json

```diff
@@ -7,12 +7,32 @@
   "author": "Edgar Andrés Margffoy Tuay <andfoy@gmail.com>",
   "license": "MIT",
   "dependencies": {
+    "@babel/core": "^7.5.5",
+    "@babel/preset-env": "^7.5.5",
+    "babel-loader": "^8.0.6",
     "fetch": "github/fetch#^2.0.3",
     "jquery": "jquery/jquery-dist#^3.2.1",
     "promise-polyfill": "taylorhakes/promise-polyfill#^6.0.2",
-    "xterm.js": "sourcelair/xterm.js#^2.7.0"
+    "xterm": "^3.14.5",
+    "xterm-addon-attach": "^0.1.0",
+    "xterm-addon-fit": "0.1.0-beta2",
+    "xterm-addon-search": "^0.1.0",
+    "xterm-addon-web-links": "^0.1.0",
+    "xterm-addon-webgl": "^0.1.0",
```

Comment:
Why is this addon needed? I mean, why do need to support webgl inside `spyder-terminal`?